src/pq_flash_index.cpp

```diff
@@ -810,7 +810,7 @@ int PQFlashIndex<T, LabelT>::load_from_separate_paths(uint32_t num_threads, cons
         }
 #endif
         parse_label_file(infile, num_pts_in_label_file);
-        assert(num_pts_in_label_file == this->_num_points);
+        assert(num_pts_in_label_file == this->num_points);
```

Comment:
why there are two "num_points"? can we make them to be one?

Reply:
there is one num_points in pqflashindex. but since its assert statement, release build didnot fail I think